core/app_modules/cal.js

```diff
@@ -21,6 +28,13 @@ module.exports.addCalendarEvent = co.wrap(function*(username, event, user){
   }
   if (!isOverlap) {
     yield client.nonQuery('INSERT INTO calendar("user", "time", room, duration, title)VALUES ($1, $2, $3, $4, $5);', [user, event.time, event.room, event.duration, event.title]);
+    var user_info = yield get_user_info(user);
+    try {
+      email.roomConfirmation(user_info, event.time, event.room)
+    }
+    catch(err){
+      console.log(err)

```

Comment:
Alternatively we could populate the calendar popup form with an email input box. That way they have a chance to fill it out if it's empty or incorrect.